src/components/user-profile-info/UserProfileInfo.tsx

```diff
@@ -63,10 +69,30 @@ const UserProfileInfo: FC<UserProfileInfoProps> = ({
     <Box sx={spliceSx(styles.root, sx.root)}>
       {renderAdditionalInfo && (
         <Link onClick={handleLinkClick} to={userURL}>
-          <Avatar
-            src={photo && `${import.meta.env.VITE_APP_IMG_USER_URL}${photo}`}
-            sx={spliceSx(styles.avatar, sx.avatar)}
-          />
+          {isOnline ? (
```

Comment:
I guess, better move this to a variable, for example onlineBadge

Reply:
Fixed